MainModule/Server/Core/Admin.lua

```diff
@@ -1244,6 +1244,58 @@ return function(Vargs, GetEnv)
 
 			Admin.PrefixCache = tempPrefix
 			Admin.CommandCache = tempTable
+
+			-- // Support for commands to be ran via TextChat
+			task.spawn(function()
+				local container = service.TextChatService.ChatVersion == Enum.ChatVersion.TextChatService and service.TextChatService:WaitForChild("TextChatCommands")
+
+				if container then
+					for _, v in container:GetChildren() do
+						if string.sub(v.Name, 1, 7) == "Adonis_" then
+							v:Destroy()
+						end
+					end
+
+					local blacklistedCommands = {}
+
+					for _, v in container:GetDescendants() do
+						if v:IsA("TextChatCommand") then
+							blacklistedCommands[v.PrimaryAlias] = true
+							blacklistedCommands[v.SecondaryAlias] = true
+						end
+					end
+
+					for name, data in Commands do
+						local command1, command2 = nil, nil
+
+						for _, v in data.Commands do
```

Comment:
![image](https://github.com/Epix-Incorporated/Adonis/assets/50679909/4f500499-13cd-4831-9646-851e083f42bf)

Reply:
I cannot reproduce this issue. Are you using any plugins?

Reply:
I just got back to this-

Anyways no I wasn't